api/src/hyp3_api/handlers.py

```diff
@@ -5,16 +5,19 @@
 
 
 def submit_job(body):
+    parameters = body['parameters']
```

Comment:
might `print(body)` here, we don't get any visibility into the body from access logs